src/lasuite/malware_detection/migrations/0004_malwaredetection_backend.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.2.3 on 2025-11-28 14:47
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('malware_detection', '0003_malwaredetection_error_code_and_more'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='malwaredetection',
+            name='backend',
+            field=models.TextField(blank=True, help_text='backend used for the detection'),
+        ),
```

Comment:
Could you put this in the same migration (0003)? The number of migration files will slowly but surely make tests longer and longer (yes we are far from any impact, but still) :)